cookbooks/vankeken_subduction/doc/vankeken_subduction.md

```diff
@@ -0,0 +1,24 @@
+# Van Keken 2008 corner flow recreation
+
+*This section was contributed by Daniel Douglas and Max Rudolph*.
+
+This cookbook recreates a 2D corner flow style subduction model as outlined in van Keken 2008.
+Since ASPECT uses a rectangular mesh by default, the geometry of this model combined with the
+way that the velocities are constrained make using ASPECT's default mesh extremely unstable when
+trying to solve for a continuous pressure field. To circumvent this issue, a custom mesh using
+quadrilaterals was designed that allows for a q2q1 discretization to become stable. The input
+files were modified only slightly from an existing PR #1864 made by Max Rudolph. Further efforts
+to confirm that this setup can more accurately reproduce the results of the 2008 benchmark should
+be done to move this from the cookbook section to the benchmark section.
+
+The model prescribes a 5cm/yr at a subduction angle of 45 degrees convergence rate along the left boundary and the interface between the subducting plate and the overriding plate and mantle wedge. In the overriding plate, the velocity is constrained to be 0. The Stokes equations are solved in the mantle wedge, and the right and bottom boundaries are open.
```

Comment:
```suggestion
The model prescribes a 5cm/yr convergence rate at a subduction angle of 45 degrees along the left boundary and the interface between the subducting plate and the overriding plate and mantle wedge. In the overriding plate, the velocity is constrained to be 0. The Stokes equations are solved in the mantle wedge, and the right and bottom boundaries are open.
```

Reply:
After this paragraph, can you add a few sentences that (i) describe what you observe when you run this scenario, and that (ii) references all figures you add?